frontend/src/components/page-connect/map-details.js

```diff
@@ -0,0 +1,39 @@
+import * as React from "react";
+import { StaticImage } from "gatsby-plugin-image";
+import { SecondaryButtonLink } from "../Button";
+
+const MapDetails = () => {
+  return (
+    <div className="flex flex-col items-center justify-center pt-5 bg-Neutral-200 md:flex-row md:h-[30.25rem] md:px-[8.125rem] md:flex-shrink-0 md:bg-[#fff]">
```

Comment:
you should not need to set height.  you should allow the other css to handle how tall it is. you can set border radius for desktop here for all four corners.  you do not need to copy figma one to one which is better to handle border radius in one place.